pkg/protocol/announcer/announcer.go

```diff
@@ -100,7 +108,8 @@ func (a *Announcer) Announce(
 	[]group.MemberIndex,
 	error,
 ) {
-	messagesChan := make(chan net.Message, a.groupSize)
+	messagesChan := make(chan net.Message, announceReceiveBuffer)
```

Comment:
We can remove the `groupSize` field from the `Announcer` struct as it is no longer used.

Reply:
Oops, I completely overlooked it. See 255aae9d8dbf2997f89e35bcb008f6d683532521, please.